### DISCUSSION TITLE
curl -s -XPOST localhost:9010/-/reload

### DISCUSSION THREAD

**Discussion by jaiswaldj** [prometheus/prometheus#9464] [Q&A]
lifecycle API is not enabled how to enable i am using this command but its not worked

**Comment by LeviHarrison** [comment]
If I'm understanding correctly, you haven't enabled the lifecycle API, but would like to know how to. You can do so by adding the `--web.enable-lifecycle` command-line flag to where you start the Prometheus server.

**Comment by jaiswaldj** [comment]
yes, I didn't enable the lifecycle API. I am adding --web. enable-lifecycle this command Prometheus. service file but it does not work. can you suggest where I put this command so I can enable this flag?
 I have used this type - 
ExecStart= /usr/local/bin/prometheus/prometheus --web.enable-lifecycle --config.file=/usr/local/bin/prometheus/prometheus.yml --storage.tsdb.path=/var/lib/prometheus/data/ --web.external-url=http://172.16.1.194:9010

**Reply by LeviHarrison** [reply]
Setting `--web.external-url=http://172.16.1.194:9010` won't the port Prometheus listens on. You have to add `--web.listen-address=:9010` to do so. Now Prometheus should be available on `localhost:9010`.

**Reply by jaiswaldj** [reply]
Thanks I have made the changes  and now service is running and lifecycle api flag also enabled but I can't access  Prometheus service in browser. through I can curl pages. I tried in different browsers but no luck

**Reply by LeviHarrison** [reply]
Are you running Prometheus on a different computer than you're using your browser on?

**Reply by jaiswaldj** [reply]
I am using my own computer and I have tried my own and different browsers both but it's not working in the browser.
